contracts/LifCrowdsale.sol

```diff
@@ -228,14 +276,15 @@ contract LifCrowdsale is Ownable, Pausable {
   }
 
   function funded() public constant returns (bool) {
-    return weiRaised >= minCap;
+    assert(weiPerUSDinICO > 0);
+    return weiRaised >= minCapUSD.mul(weiPerUSDinICO);
   }
 
   // return the eth if the crowdsale didnt reach the minCap
   function claimEth() public {
     require(isFinalized);
     require(hasEnded());
-    require(funded());
+    require(!funded());
```

Comment:
Why is this changed to `!funded`? I think it should stay `funded`, eth can be claimed by the foundation only if the min cap was reached, right?

Reply:
This function is not for the foundation, its for the investors. If the minCap wasn't reached they can claim their ether back.

Reply:
Oh, right! I was confusing the MarketMaker here :)
We have to add a claimEth command to the property-based test, it's obvious that this is not being tested because it should have made the tests to fail

Reply:
:+1: going to create an issue for this